### DISCUSSION TITLE
Custom types handling (enum, postgres)

### DISCUSSION THREAD

**Discussion by arnaud** [directus/directus#13090] [Feature Requests]
When a table has one or more fields with custom types (enums?), Directus can't even list the table fields/columns.

### To Reproduce

In a postgres database:

- Define a custom type:

    ```sql
    DROP TYPE IF EXISTS public.WOW;
    CREATE TYPE public.WOW AS ENUM ('amazing', 'marvelous', 'incredible');
    ```

- Have a pre-existing table with one field relying on this custom type:

    ```sql
    CREATE TABLE public.wonderful (
      name VARCHAR(60)
    , aint_it_amazing public.WOW NOT NULL
    );
    ```

**Comment by azrikahar** [comment]
Currently there's no native support for custom types (or enums) yet, but this has been brought up (primary related to enums) in #4887 and #7985 👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
